roles/static_inventory/templates/inventory.j2

```diff
@@ -44,6 +44,7 @@ masters.{{ stack_name }}
 
 [etcd:children]
 etcd.{{ stack_name }}
+{% if groups['etcd']|length == 0 %}masters.{{ stack_name }}{% endif %}
```

Comment:
I would think so, but it's not the case. This is the error I was getting without it:

```
TASK [static_inventory : create inventory from template] ***********************
Thursday 20 July 2017  09:14:42 +0000 (0:00:00.026)       0:01:37.329 ********* 
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'etcd'"}
```

https://travis-ci.org/tomassedovic/openshift-ansible-contrib/jobs/255581702#L894